doc/manual/api.md

```diff
@@ -16,35 +16,68 @@ async def example(room, message):
     example_message = "Hello World"
     if match.is_not_from_this_bot():
         await bot.api.send_text_message(
-            room_id=room.room_id, 
+            room_id=room.room_id,
+            message=example_message)
+```
+The first two arguments are required. The room_id argument is the id of the destination room. The message argument is the string that is to be sent as a message.
+
+#### Sending a bot notice
+To send a bot notice (non-alerting message), set the argument `msgtype="m.notice"`. It can be combined with any of the other optional arguments.
+```python
+async def example(room, message):
+    match = botlib.MessageMatch(room, message, bot)
+    example_message = "Hello World"
+    if match.is_not_from_this_bot():
+        await bot.api.send_text_message(
+            room_id=room.room_id,
             message=example_message,
             msgtype="m.notice")
 ```
-The first two arguments are required. The room_id argument is the id of the destination room. The message argument is the string that is to be sent as a message. The msgtype argument can be "m.text" (default) or "m.notice".
 
-### Using the send_image_message method
-The send_image_message method of the Api class can be used to send image messages in Matrix rooms. An example is shown in the following python code.
+#### Sending a reply
+To send your message as a reply to another message, pass the original message event to the `reply_to` argument. It can be combined with any of the other optional arguments.
 ```python
 async def example(room, message):
     match = botlib.MessageMatch(room, message, bot)
-    example_image="./img/example.png"
+    example_message = "Hello World"
     if match.is_not_from_this_bot():
-        await bot.api.send_image_message(
-            room_id=room.room_id, 
-            image_filepath=example_image)
+        await bot.api.send_text_message(
+            room_id=room.room_id,
+            message=example_message,
+            reply_to=message)
```

Comment:
I think calling the received event "`message`" might be confusing here, since the parameter is also called `message` and the text is `example_message`. Perhaps rename it to `event`?

Reply:
Renaming it to either `event_content` or `message_content` would probably be better. I think that `event` would be confusing.

Reply:
it's not content though, it's the actual event, isn't it?

Reply:
You are right. What about content_body?